style.css

```diff
@@ -0,0 +1,94 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.calculator {
+  border: 1px solid #999999;
+  border-radius: 6px;
+  box-shadow: 0px 15px 20px 5px rgba(136, 136, 136, 0.5);
+  height: 318px;
+  left: 50%;
+  position: absolute;
+  top: 25%;
+  width: 231px;
+}
```

Comment:
going to merge this to master and come back to it because i have been struggling with it for longer than i should be. Unless you're free for me to come and find you @deadlyicon